api/adapters/hydroshare.py

```diff
@@ -0,0 +1,260 @@
+import requests
+from starlette import status
+from datetime import datetime
+from typing import List, Optional, Union
+from pydantic import BaseModel, EmailStr, HttpUrl
+from api.adapters.base import AbstractRepositoryMetadataAdapter, AbstractRepositoryRequestHandler
+from api.exceptions import RepositoryException
+from api.models import schema
+from api.models.catalog import DatasetMetadataDOC
+
+
+class _HydroshareRequestHandler(AbstractRepositoryRequestHandler):
```

Comment:
I don't think the _ at the beginning of the class is "pythonic".  What is your intention for using it?

Reply:
This is a private class and not supposed to be imported.